# Group Report

## Section 1 (Sophia)
- The authors are attempting to increase atomic accuracy to address the gaps within single protein structures, the “protein folding problem.” They developed the first computational approach that is able to predict protein structures near experimental accuracy. They entered their neural network, AlphaFold, into the CASP14 assessment (protein prediction center), and crushed the competition in major protein prediction. AlphaFold’s predictions were off by only 0.96 Ångströms (unit to measure the size of atoms) on average (aka less than 1/10th the width of a single atom). The second-best method was off by 2.8 Ångströms—nearly three times worse. AlphaFold achieved a level of accuracy that was previously thought impossible: it can predict the protein backbone and also the precise positions of side chains.
  
## Section 2 (Leeva)
- The authors decided to use computational tools that focused either on the protein’s physical interactions or the evolutionary history to determine its three-dimensional structure, with physical interactions focusing more on the proteins properties and mechanisms and the evolutionary history is how the protein is believed to have evolved. This was developed with assistance from the Protein Data Bank (PDB) and genomic sequencing data, but even with these methods they fall short of experimental results. Evoformer shows protein structure predictions, and how certain residues in the protein statistically and geometrically interact with each other. An IPA method was used to combine single, pair, and geometric representations in 3D space to predict protein structure and how each residue interacts with one another. Imputed networks include the amino acid sequence of the protein of interest as well as the amino acid sequence of evolutionarily related proteins.

## Section 3 (Haider)
- The paper relies on various references, all of which are publicly available. The paper compares against protein structures in the past that have been deposited in the Protein Data Bank, which is a library filled with such completed structures. This is to verify the accuracy of the protein structures predicted using AlphaFold. UniRef90, Uniclust30, BFD, and MGnify are databases with sequences of proteins. These libraries are used to predict the structures of proteins by comparing the sequence of the protein with the sequences of its cousin proteins. The paper shows that removing BFD or MGnify measurably reduces accuracy, proving these sources are critical. The paper also references CASP, which is a blind community test in which teams submit the 3D structures of solved proteins that the organizers know the actual 3D structures of but have not released to the public. This is important because it is a blind test, and AlphaFold2’s showed significantly greater accuracy than other systems. The goal is defined by Anfinsen’s idea that a protein’s sequence determines its 3D shape, as well as Dill’s review on why predicting that shape is hard because very subtle factors change what the shape would look like. The paper of course also references the prior AlphaFold because that is what the new AlphaFold model is built on. The paper also relies on deep-learning “contact prediction” papers (2017–2019). These papers are about showing neural nets can spot which parts of a protein touch, which is important key groundwork for structure prediction. Finally, the study relies on standard accuracy scores (lDDT, GDT, and TM-score) so everyone can compare results on the same scale. 

## Section 4 (Rayhan)
- There are many reasons why we will not be able to replicate the study’s findings using their GitHub repository. We do not have the hardware or computing power to run Alphafold. The full installation of Alphafold requires up to 3 TB of disk space to keep genetic databases and a modern NVIDIA GPU. The disks should also be SSD and the NVIDIA GPU must have a large enough amount of memory and CUDA which contains 8-series family of GPUs from NVIDIA or later and even then some CUDA versions are not compatible with Docker or Alphafold. We are all using personal computers which are not as powerful as the cloud servers and research clusters that have much more power which is necessary to run Alphafold properly. We also do not have the right environment. We need to use Linux and Docker because Alphafold does not support other OSes. We will also need extra databases and model parameters in a separate directory such as aria2c,  BFD, MGnify, PDB70, PDB (structures in the mmCIF format), PDB seqres – only for AlphaFold-Multimer, UniRef30 (FKA UniClust30), UniProt – only for AlphaFold-Multimer, and  UniRef90. Also because the databases are updated we we would not get the same results as those shown in CLASP14 as the data has changed.
